### ISSUE TITLE
[Off topic] spam

### ISSUE THREAD

**Issue by sjehuda** [OpenUserJS/OpenUserJS.org#1997]
See https://openuserjs.org/users/joker/comments

**Comment by sjehuda** [comment]
See https://openuserjs.org/users/joker/comments

Closing

**Comment by sjehuda** [comment]
Please ignore this message.

I was wrongly judgin by two factors:

- User agent as used by previews deleted profile which is actually a common one. `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36`
- URLs freebitco.in photolessons.org

I'll use the [flag](https://openuserjs.org/scripts/joker/LiteKing_Faucet_Clicker/issues/How_to_use_script#comment-18805db3992) button next time.

Sorry for the misunderstanding.

**Comment by Martii** [comment]
An additional factors you should consider:

1. Does the script take you to photoloessons.org without your permission/knowledge. This is a rhetorical question... it currently does not. In other words the target site "could be" malicious in nature however that doesn't make the .user.js spam. If you want to warn other users about a possible bad site you may with some tact. However using a very short label without citation like you did is very bad. If you think the script needs mitigation from Moderators and up then flag the Script itself. Comments are currently done by flagging the User and then referencing what you are talking about with a brief reason why you think it should be evaluated.

> User agent as used by previews deleted profile which is actually a common one.

Dare I ask what you mean here? Again the .user.js script is not currently to fault here no matter how you possibly explain this.

**Comment by sjehuda** [comment]
On Wed, 10 May 2023 07:58:42 -0700
Marti Martz ***@***.***> wrote:

> > User agent as used by previews deleted profile which is actually a
> > common one.  
> 
> Dare I ask what you mean here? Again the .user.js script is not
> currently to fault here no matter how you possibly explain this.

I was referring  user agent used by the author.

In recent even when I publicized a script of mine, I've recorded the
User Agent of the commenter (which probably was a spam user/bot) which
was the same to the author I have commented on.

P.s.

Later checking, that UA is the most popular in the list of user agents
I've published in one of my libraries.

So I made a reckless connection.
